app/views/users/logix/index.html.erb

```diff
@@ -29,10 +29,12 @@
     </div>
   <% end %>
 
+
   <div class ="d-flex flex-column align-items-center flex-md-row justify-content-md-between align-items-md-center">
     <div class="d-flex flex-row justify-content-center align-items-center">
-      <%= image_tag @user.profile_picture.url(:medium, alt: "@user.name"), style: "width: 100px; height: 100px; border: 0px;" %>
+      <%= image_tag @user.profile_picture.url(:medium, alt: "@user.name"), style: "width: 100px; height: 100px; border: 0px; margin: 10px; object-fit: cover; border-radius: 50%;" %>
```

Comment:
Move to stylesheet or use bootstrap classes to accomplish the same

Reply:
Done